libs/template/experimental/virtual-scrolling/src/lib/scroll-strategies/autosize-virtual-scroll-strategy.ts

```diff
@@ -290,9 +290,7 @@ export class AutoSizeVirtualScrollStrategy<
   ): void {
     this.viewport = viewport;
     this.viewRepeater = viewRepeater;
-    this.resizeObserver = new ResizeObserver((events) => {
-      this.viewsResized$.next(events);
-    });
+    this.resizeObserver = new RxaResizeObserver();
```

Comment:
Why not using DI instead of manual instantiation?

Reply:
would be possible, yes. Do u have an issue with skipping this for a next PR?